src/main/java/ome/services/util/DBPatchCheck.java

```diff
@@ -34,7 +34,7 @@ public DBPatchCheck(SqlAction sql, PreferenceContext prefs) {
     private final static String line = "***************************************************************************************\n";
     private final static String see = "See https://docs.openmicroscopy.org/latest/omero/sysadmins/server-upgrade.html\n";
     private final static String no_table = mk("Error connecting to database table dbpatch. You may need to bootstrap.\n");
-    private final static String wrong_version = mk("DB version (%s) does not match omero.properties (%s). Please apply a db upgrade.\n");
+    private final static String wrong_version = mk("DB version (%s) does not match omero-common.properties (%s). Please apply a db upgrade.\n");
```

Comment:
Ah, "end users" won't be working with a combined file generated from the build? Okay.

Reply:
I made it more explicit so people know where to look at.
Actually it is now in omero.model